From: jbeulich@novell.com
Subject: xenbus_be currently doesn't build properly as a module

It depends on the non-exported symbols vmlist and vmlist_lock.

Index: head-2006-04-20/drivers/xen/xenbus/Makefile
===================================================================
--- head-2006-04-20.orig/drivers/xen/xenbus/Makefile	2006-04-20 10:12:48.000000000 +0200
+++ head-2006-04-20/drivers/xen/xenbus/Makefile	2006-04-20 13:52:56.000000000 +0200
@@ -1,5 +1,6 @@
 obj-y	+= xenbus.o
-obj-$(CONFIG_XEN_BACKEND) += xenbus_be.o
+#obj-$(CONFIG_XEN_BACKEND) += xenbus_be.o
+obj-y   += xenbus_be.o
 
 xenbus_be-objs =
 xenbus_be-objs += xenbus_backend_client.o
